torch/nn/modules/batchnorm.py

```diff
@@ -500,8 +500,8 @@ def convert_sync_batchnorm(cls, module, process_group=None):
                                                    process_group)
             if module.affine:
                 with torch.no_grad():
-                    module_output.weight.copy_(module.weight)
-                    module_output.bias.copy_(module.bias)
+                    module_output.weight = module.weight
```

Comment:
out of curiosity, instead of copying member by member, can you do `load_state_dict` thing here that you do in the tests?

Reply:
Nope. `load_state_dict` is basically a `copy_`. So it would not get the device property appropriately.
It worked in the test case because we explicitly moved the module to cuda to match the original.